docs/websites.md

```diff
@@ -155,14 +148,15 @@ functions:
 
 plugins:
     - ./vendor/bref/bref
-    # This lets us use `#{Assets.Arn}` variables
-    - serverless-pseudo-parameters
 
 resources:
     Resources:
         # The S3 bucket that stores the assets
         Assets:
-            # [...] see the previous section for details 
+            Type: AWS::S3::Bucket
+            Properties:
+                BucketName: <bucket-name>
+                AccessControl: PublicRead # needed for the cdn to read the directory structure
```

Comment:
Could you explain why we need to add this?

Do you mean that this is required specifically by the `S3OriginConfig` integration with CloudFront? I am surprised we don't need it otherwise.

Reply:
I was confused too. But without it, the cdn only delivers access denied. But I also can't remove the `AssetsBucketPolicy` because then i also get access denied when accessing resources.

I can investigate further if you want, the official way is to create an s3 origin identity so that only the cdn can access files. I tried that once but it took an irritatingly long time before everything worked.

Reply:
Interesting,

I investigated further and noticed that it makes absolutely no difference.
I'm a bit confused why i needed it before... but yes it works.

I found this template from aws: https://github.com/awslabs/aws-cloudformation-templates/blob/5c66bbfaec08313fcfee48b49ce6ba0a38f6bb1a/community/solutions/StaticS3CloudFront.yml
It uses an s3 origin identity to restrict access to the bucket to the cdn but there is also no ACL configuration.

So i'll remove it. Good that you asked ;)